aspnetcore/migration/mvc/sample/WebApp1.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="wwwroot\css\site.css" />
+    <Content Include="wwwroot\css\site.min.css" />
+    <Content Include="wwwroot\favicon.ico" />
+    <Content Include="wwwroot\js\site.js" />
+    <Content Include="wwwroot\js\site.min.js" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="BundlerMinifier.Core" Version="2.3.327" />
```

Comment:
@spboyer @scottaddie I removed the bundle/minify section of this tutorial in favor of linking to the bundle/minify doc but missed removing the package reference from the sample project.  I propose to do that (remove this package reference) rather than change it to a tools reference.  There are still unresolved issues with bundling/minification (see #1786 and https://github.com/madskristensen/BundlerMinifier/issues/230). Thoughts?